community/athens-guide/feature-list/backups.md

```diff
@@ -0,0 +1,29 @@
+# Backups in Athens Research 
+
+At Athens, we fundamentally believe in data ownership and data privacy. It is from this belief that our backup system has been implemented. The primary goal of our backup system is to make it possible to revert your data back to any point in time so that there is  *zero* room for data loss due to corruption and other formatting issues. 
+## Restoring from Backups
+Backups in Athens are stored in a .bkp format which is easy to restore from. They are named with a unix timestamp, although the Restoring is a 5 step process. 
+#### Locate your database folder
+Backups in Athens are stored in your main db folder and have a .bkp file extension. To locate your database folder, in the Athens app, click the database icon on the toolbar and you will see the current location of your database. In this folder, you will see your backup files. 
+<img width="1552" alt="Screen Shot 2021-06-27 at 22 18 17" src="https://user-images.githubusercontent.com/80150109/123552819-954ecb00-d795-11eb-8d86-eb2e246737b8.png">
+
+#### Relocate your index.transit
+Before we proceed to the next step, locate your current index.transit file and relocate it to another folder. 
+![](https://user-images.githubusercontent.com/80150109/123552515-160cc780-d794-11eb-961d-8c277b3f632e.gif)
+#### Locate the backup file you want to restore from
+Once you are in the folder, find a file with a creation time that corresponds to the state of your database you would like to revert to. For instance, let's say I want to restore my DB to where it was on the night of 6th june. In this case, I would scroll until I find a bkp file with a creation time around that specific time. 
```

Comment:
Let's convert this to being in second person, so it's talking about "you" instead of "I."

I also added mention of converting Unix time in case there's confusion around the creation time, which presents itself different on every OS.

```suggestion
Once you are in the folder, find a file with a creation time that corresponds to the state of your database you would like to revert to. For instance, let's say you want to restore my DB to where it was on the night of June 6th. In this case, you find a `.bkp` file with a creation time around that specific time. You can also use a [Unix time converter](https://time.is/Unix_time_converter) to figure out when Athens created a `.bkp` file.
```